### PR TITLE
Two minor CMake changes

### PR DESCRIPTION
**Pull request by tstenner**
1. Use the default CMake install directories from `GNUInstallDirs`
2. Try to use the system pugixml first if it exists, compile in the bundled pugixml otherwise

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -25,18 +24,31 @@ set(SOURCES
...
+find_package(pugixml QUIET)
+if(TARGET pugixml AND NOT XDF_NO_SYSTEM_PUGIXML)
+	message(STATUS "Using system pugixml")
+	target_link_libraries(xdf PRIVATE pugixml)
+else()
+	message(STATUS "Using bundled pugixml")
+	target_sources(xdf PRIVATE pugixml/pugixml.cpp)
+	target_include_directories(xdf PRIVATE pugixml)
+endif()
+
```
Comment (Yida-Lin) [inline CMakeLists.txt]: This looks a bit risky; not sure how much difference between different pugixml versions, but things might break.

Comment (tstenner) [reply CMakeLists.txt]: Differences in minor versions should be unproblematic and this is mostly aimed at linux distributions that will patch in something like this anyways. I'll add a minimum version requirement, just to be on the safe side.

Comment (Yida-Lin) [reply CMakeLists.txt]: Sounds good.

Comment (tstenner) [reply CMakeLists.txt]: Added in d7eb241

**Comment by tstenner** [comment]
I've added the pugixml folder to the qmake include path so now it builds again.

**Pull request merged**